### DISCUSSION TITLE
Export Custom Entities?

### DISCUSSION THREAD

**Discussion by timtutt** [winkjs/wink-nlp#83] [Help on coding] [answered]
Is it currently possible to export a model built by using `learnCustomEntities()`. It generally takes a few seconds for some of the models we've built and I'd love to just store that and reload it in the future rather than "re-training". Is there an API for this that I'm missing?

**Comment by rachnachakraborty** [comment, marked as answer]
Hey @timtutt 

Thanks for exploring the custom entities in winkNLP.  The feature to export custom entities and load as and when needed definitely makes sense. We shall add this to our Roadmap and keep you posted.

You may consider contributing to development of  APIs for enhanced usage.

Best,
Rachna
